bundle/language_en.properties

```diff
@@ -2001,12 +2011,12 @@ angal.ward.telephone
 angal.ward.telephoneedit                                                         = Telephone
 angal.ward.wardwithpharmacy                                                      = Ward with pharmacy
 angal.wardpharmacynew.selectward                                                 = Select a ward
-angal.wardpharmacynew.ward                                                       = Another ward
+angal.wardpharmacynew.ward                                                       = Another ward'
```

Comment:
is the apostrophe intentional?

Reply:
Thanks; too many project; working too fast, and not enough sleep :(